docs/source/roadmap.md

```diff
@@ -0,0 +1,30 @@
+# Roadmap
+
+The roadmap for NeuroBlueprint
+contains our development plans for the package.
+It is closely aligned with
+[datashuttle roadmap]()
+and our
+[guiding principles]().
+
+The roadmap is responsive to the requirements
+of the community and we encourage people to get in
+contact to outline their needs and priorities.
+
+The aim of NeuroBlueprint v2.0 was to
+provide a lightweight specification
+enabling automatic discovery of raw data and
+derivatives for different data types (e.g., behavior, electrophysiology).
+
+Next, we plan to extend the specification to support automated
+multimodal analysis. This will require metadata standardisation,
+and we will decide on an existing metadata standard to adopt and minimal
+set of required keywords.
+
+**Q3 2024**
+
+- Investigate and adopt a metadata standard (e.g. BIDS or Allen AIND)
```

Comment:
I would expand on the acronyms and provide links to the spec.

Reply:
In general this point is slightly strange because we mention BIDS alignment but then indicate we might go to the Allen metadata standard. This is a discussion to have though and I don't know much about the merits of each system or how well they might interoperate. Is it still worth including this here?

Reply:
I think it's worth including. I assume we will always need something on top of BIDS metadata anyway, AFAIK it doens't include everything we would need for automated analyses.